packages/boomerang/boomerang.1.1.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+name: "boomerang"
+version: "1.1.0"
+maintainer: "root <root@solomon-X556UA>"
+authors: "root <root@solomon-X556UA>"
```

Comment:
These do not appear to be a valid email address. We do prefer having some point of contact; if you do not want your full name on there then a GitHub handle is sufficient.